infra/bridge/src/hooks/twitter.js

```diff
@@ -8,8 +8,9 @@ const logger = require('./../logger')
 const oauth = new OAuth(
   'https://api.twitter.com/oauth/request_token',
   'https://api.twitter.com/oauth/access_token',
-  process.env.TWITTER_CONSUMER_KEY,
-  process.env.TWITTER_CONSUMER_SECRET,
+  process.env.TWITTER_WEBHOOKS_CONSUMER_KEY || process.env.TWITTER_CONSUMER_KEY,
```

Comment:
nit: there are a couple of places this check is performed. It might be more safe if there would be a couple of util functions e.g. `getTwitterConsumer[Key/Secret]` where this check would be done. Quickly you can change something in one place and forget to do it in another.

Reply:
Yeah, that sounds good